openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -37,6 +36,7 @@ def get_api_client(api_config, student):
     return EdxRestApiClient(api_config.internal_api_url, jwt=id_token)
 
 
+# TODO: Move to program utilities. Will require updating tests.
```

Comment:
I think doing it as part of this PR would increase the scope unnecessarily. My plan is to do this in a separate PR, to keep things clean and avoid slowing down the more immediately useful parts of this PR. Let me know if you feel strongly about doing it now.